<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Transition</class>
 <widget class="QDialog" name="Transition">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>367</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Solicitação de transição</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="requestCaptcha">
     <property name="text">
      <string>Gerar captcha</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Insira o captcha mostrado na catraca:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="captchaInserted"/>
   </item>
   <item>
    <widget class="QPushButton" name="verifyCaptcha">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Validar código</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextBrowser" name="textBrowser"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
